test/fixtures/dialects/postgres/postgres_lock_table.sql

```diff
@@ -0,0 +1,2 @@
+LOCK TABLE films IN SHARE MODE;
+LOCK TABLE films IN SHARE ROW EXCLUSIVE MODE;
```

Comment:
can we add a few more test cases here?

Reply:
Those are the one ones in [the Postgres docs](https://www.postgresql.org/docs/current/sql-lock.html).

While I'd like to add more, I'm a little hesitant to do that, without being able to test them (I don't use Postgres) and locking in any incorrect assumptions I make. Bad that I have those assumptions in the grammar, but worse (IMHO) is we add a test case to further cement this.

Reply:
Added a few more examples I managed to cobble from the internet. It's still not fully covering all the syntax I've added - seems there's a lot more syntax support than most people use!